version

```diff
@@ -1 +1 @@
-1.34a
\ No newline at end of file
+2
```

Comment:
Changed to version 2 since there so much change vs original and to fix the notification that ibramenu show saying that it is not up to date.